### ISSUE TITLE
Create a web and app interface to Eco chat

### ISSUE THREAD

**Issue by johnkslg** [StrangeLoopGames/EcoBugBounties#5]
We have a slack-like interface for Eco chat, this bounty requires starting a companion app and web code to participate in that chat from outside the game.  This will allow players to get notifications about the Eco world on their phone, including push notifications.

**Comment by nicholasguyett** [comment]
Which browsers are being supported?

Which mobile platforms do we want to target?

**Comment by noblackthunder** [comment]
@nicholasguyett The anwser to that is most likely all modern systems, meaning one the browser site : 
Edge, Safari, Firefox , Chrome etc.
on mobile it would be the 3 major platfomrs, Iphone, Android and windows phone.

**Comment by noblackthunder** [comment]
there is a limited api in the game to get some info out. but people need to build it them self. this requires dev tier of the game where you get access to the source code to add this

**Comment by MightySCollins** [comment]
Yep that's what I thought, looking though it and there did not seem to be anything related to chat which was slightly annoying.
If I understand it the task is quite large:

- Build API for Chat messages
- Build Web Interface for chat
- Make iOS/Android App's
